scripts/in_container/_in_container_utils.sh

```diff
@@ -394,9 +394,9 @@ function verify_suffix_versions_for_package_preparation() {
 
     if [[ ${VERSION_SUFFIX_FOR_SVN} =~ ^rc ]]; then
         echo
-        echo "${COLOR_RED_ERROR} The version suffix for SVN is used only for file names in RC version  ${COLOR_RESET}"
+        echo "${COLOR_YELLOW_WARNING} The version suffix for SVN is used only for file names not for versions in RC version in SVN${COLOR_RESET}"
```

Comment:
This line is a bit unclear, especially "used only for file names not for versions in RC version in SVN"

Reply:
True. Fixed.